tox.ini

```diff
@@ -31,6 +31,10 @@ commands = pylint -f colorized -E google test --ignore=src-gen
 deps = pylint
 commands = pylint -f colorized -d all -e W -r n google test --ignore=src-gen
 
+[testenv:pylint-no-test-deps]
+deps = pylint
```

Comment:
Fixed build errors by removing linuxbrew install altogether; instead, switched to depending on the `grpcio-tools` Python package to provide protoc.